Kickstarter-iOS/Features/RewardsCollection/Controller/WithShippingRewardsCollectionViewController.swift

```diff
@@ -193,11 +193,10 @@ final class WithShippingRewardsCollectionViewController: UICollectionViewControl
 
     self.viewModel.outputs.rewardsCollectionViewIsHidden
       .observeForUI()
-      .observeValues { [weak self] shimmerLoadingViewIsHidden in
+      .observeValues { [weak self] rewardsCollectionViewIsHidden in
         guard let self else { return }
 
-        self.collectionView.isHidden = !shimmerLoadingViewIsHidden
-        self.collectionView.layoutIfNeeded()
+        self.collectionView.isHidden = rewardsCollectionViewIsHidden
```

Comment:
Making sure the intent was to invert this isHidden check

Reply:
Yeah, I found the naming to be super confusing before, so I've inverted it (both here and in the view model) to be more intuitive. It's still in sync with the underlying value; it's just the output signal value that's inverted. Thanks for checking though!